src/target/linux.rs

```diff
@@ -141,6 +143,12 @@ impl LinuxTarget {
             })
             .collect())
     }
+
+    /// Kill debuggee when debugger exits.
+    fn kill_on_exit(&self) -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
I think this can be a `pub fn` for the case when a user wants to use this option after `attach`, but it's unclear whether this should be a function call, or a configuration parameter like `target.attach(pid, Config { kill_on_exit: true, ... })`. Let's leave it as it is now & create an issue for future reference.